### ISSUE TITLE
Wrong main tag

### ISSUE THREAD

**Issue by eberhapa** [KriesiMedia/Enfold-Feature-Requests#106]
It seems that the theme adds the <main>-Tag only and always to the first section.

We had an report made by an accessibility company and they said that the main tag should be on the div with the id main that is above all the content.
